stepaction/tekton-catalog-publish/0.1/README.md

```diff
@@ -0,0 +1,110 @@
+# Tekton Catalog Publish
+
+This stepaction publishes each Task/StepAction in a Tekton catalog as [Tekton Bundles](https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles).
+The catalog must be structured as a [Tekton Catalog](https://github.com/tektoncd/catalog#catalog-structure).
+
+Bundles are published to `$REGISTRY/$PATH/<task/stepaction-name>:<task/stepaction-version>` and, when `TAG` is provided, to
+`$REGISTRY/$PATH/<task/stepaction-name>:$TAG`. An example of extra tag could be the git sha of the catalog repo that
+is being published. The task uses the `tkn bundle` command to publish, which is available
+in `tkn` starting with version v0.18.0.
```

Comment:
we should hold on right now with this as currently tkn doesn't support stepactions
cc @piyush-garg @vdemeester

Reply:
I think we just need the `tkn bundle push` command here. I don't think it cares what is in the bundle (at least I don't think so) which is why the CI tests pass. So technically to run this command, support for Step actions is not needed. I might be completely wrong here though.